composer.json

```diff
@@ -49,6 +49,7 @@
         "ocramius/package-versions": "^1.9",
         "php-parallel-lint/php-parallel-lint": "^1.2",
         "phpstan/phpstan-nette": "^0.12.12",
+        "phpstan/phpstan-webmozart-assert": "^0.12.8",
```

Comment:
added `phpstan/phpstan-webmozart-assert` to require-dev to handle Assert::isA/AnyOf